stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -378,10 +518,10 @@ extension Unsafe${Mutable}BufferPointer: ${Mutable}Collection, RandomAccessColle
 %end
   }
 
-  // Skip all debug and runtime checks
-
-  @inlinable // unsafe-performance
+  @_spi(SwiftStdlibLegacyABI) @available(swift, obsoleted: 2)
+  @usableFromInline
   internal subscript(_unchecked i: Int) -> Element {
+    @_transparent
```

Comment:
Transparent for only the stdlib? I'm not sure this implementation is visible from the outside because of `@usableFromInline`.

Reply:
Oh, I accidentally kept this from a previous version where this remained public. The attribute does nothing for obsoleted functions, so there is no point to keeping it.